server/src/test/java/org/elasticsearch/snapshots/DedicatedClusterSnapshotRestoreIT.java

```diff
@@ -418,9 +419,9 @@ public void testSnapshotDuringNodeShutdown() throws Exception {
         SnapshotInfo snapshotInfo = waitForCompletion("test-repo", "test-snap", TimeValue.timeValueSeconds(60));
         logger.info("Number of failed shards [{}]", snapshotInfo.shardFailures().size());
         logger.info("--> done");
+        BlobStoreTestUtil.assertRepoConsistency(internalCluster(), "test-repo");
```

Comment:
instead of adding this to the end of each test, can't we just do this with an `@After`?

Reply:
Can't do that unfortunately since we have some tests that leave the repo in a different broken state (I will explain below in a sec, but same goes for the other tests I added this to). Also, some tests use multiple repositories => adding it manually seemed easier than adding some other logic to keep track of what repos were created etc.

Reply:
should we instead have those tests opt out of the check behavior perhaps? It creates less noise and is perhaps more specific in what remains to be addressed.

Reply:
I guess we could do that => let me see what I can do there, on it :)

Reply:
How about af47bb8 ? :) Seems my memory didn't serve me well here and we actually only have a single test that leave the repo in broken (and not yet cleaned up state). :)

Reply:
> How about af47bb8 ?

I like this better.